assets/css/main.css

```diff
@@ -3,13 +3,152 @@
   -moz-box-sizing: border-box;
   box-sizing: border-box; }
 
-.container {
+h1, h2, h3, h4, h5, h6,
+ul, li,
+p {
+  padding: 0;
+  margin: 0; }
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: normal; }
+
+body {
+  background-color: #450081;
+  color: #ffffff;
+  font-family: Lato, Arial, sans-serif; }
+  body a {
+    color: #ffffff;
+    text-decoration: none; }
+
+.disclaimer {
+  margin: 0 auto 0px auto;
+  padding: 10px;
+  text-align: center;
+  background-color: #f5f5f5;
+  color: #000000; }
+
+header {
+  max-width: 68em;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px 0; }
+  header:after {
+    content: "";
+    display: table;
+    clear: both; }
+  header h1 {
+    float: left;
+    display: block;
+    margin-right: 2.35765%;
+    width: 74.41059%;
+    margin-top: 7px;
+    padding-left: 20px; }
+    header h1:last-child {
+      margin-right: 0; }
+  header ul {
+    float: left;
+    display: block;
+    margin-right: 2.35765%;
+    width: 23.23176%;
+    max-width: 68em;
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 0; }
+    header ul:last-child {
+      margin-right: 0; }
+    header ul:after {
+      content: "";
+      display: table;
+      clear: both; }
+    header ul li {
+      float: left;
+      display: block;
+      margin-right: 2.35765%;
+      width: 31.76157%;
+      margin-top: 17px; }
+      header ul li:last-child {
+        margin-right: 0; }
+
+hr.white {
+  height: 2px;
+  background-color: #ffffff; }
+
+.content {
   max-width: 68em;
   margin-left: auto;
-  margin-right: auto; }
-  .container:after {
+  margin-right: auto;
+  margin-left: 17.05961%; }
+  .content:after {
     content: "";
     display: table;
     clear: both; }
+  .content h1 {
+    float: left;
+    display: block;
+    margin-right: 2.35765%;
+    width: 65.88078%;
+    text-align: center;
+    font-size: 80px;
+    margin-bottom: 10px; }
+    .content h1:last-child {
+      margin-right: 0; }
+  .content h2 {
+    float: left;
+    display: block;
+    margin-right: 2.35765%;
+    width: 65.88078%;
+    text-align: center;
+    font-size: 36px;
+    margin-top: 20px;
+    margin-bottom: 20px; }
+    .content h2:last-child {
+      margin-right: 0; }
+  .content ul.results {
+    float: left;
+    display: block;
+    margin-right: 2.35765%;
+    width: 65.88078%;
+    max-width: 68em;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 40px;
+    margin-bottom: 40px; }
+    .content ul.results:last-child {
+      margin-right: 0; }
+    .content ul.results:after {
+      content: "";
+      display: table;
+      clear: both; }
+    .content ul.results li {
+      float: left;
+      display: block;
+      margin-right: 2.35765%;
+      width: 48.82117%;
```

Comment:
We'll want to use the NEAT grid system mixins here, instead of explicitly stating these percentages.